Utils/should_trigger_test_upload.sh

```diff
@@ -8,7 +8,7 @@ IGNORED_FILES=(
     "Tests/conf.json"
     "Tests/known_words.txt"
     "Utils/should_trigger_test_upload.sh"
-    "Tests/tests_end_to_end_xsiam/README.md"
+    "Tests/tests_e2e/content/xsiam/README.md"
```

Comment:
why? and if keeping it, why not add the XSOAR SAAS file as well?

Reply:
e2e of XSIAM is only triggered in upload, while e2e of xsoar-saas is in nightly
hence there is no reason to add xsoar-saas as well as it doesn't get executed in upload.

Reply:
correction: i just see its the `test-upload` so you are right, ill add xsoar-saas as well.